src/crosscat_utils.py

```diff
@@ -47,6 +51,12 @@ def draw_crosscat(bdb, generator, modelno, row_label_col=None):
         Name of generator.
     modelno: int
         Number of model to draw.
+    row_label_col: str
+        Name of column to use for row labels
+    kwargs: ???
```

Comment:
rng should canonically come before **kwargs. "which see" is incomplete, but I presume you wanted to say something like "for which, see help(bdbcontrib.crosscat_utils.draw_state)"?

If so, why now document row_label_col here, rather than there, seeing as you're removing it from the explicit arg list?